libs/data-access/http-requests/src/lib/data-access-http-requests.ts

```diff
@@ -41,3 +43,37 @@ export const getAdverts = () => {
   });
   return response;
 };
+
+export const create = ({ data, table }) => {
```

Comment:
What do you mean 'table' ? same bellow

Reply:
"table" - adverts table inside DB. Exmaple. POST: "/api/adverts/", so it saves advert in Adverts table.